commons-civil/src/main/java/uk/gov/hmcts/reform/civil/documentmanagement/SecuredDocumentManagementService.java

```diff
@@ -184,6 +185,39 @@ public byte[] downloadDocument(String authorisation, String documentPath) {
         }
     }
 
+    @Retryable(value = DocumentDownloadException.class, backoff = @Backoff(delay = 200))
+    @Override
+    public DownloadedDocumentResponse downloadDocumentCUI(String authorisation, String documentPath) {
```

Comment:
Sorry Leo for looking at your pr and then commenting on it. In my opinion - civil service should not know anything about CUI or who is calling it. Perhaps the method name should  be changed to something more meaningful avoiding a client name that would be calling it (because what if we use this somewhere else too not just cui)

Reply:
updated to reflect that its focusing on getting the metadata too, renamed to `downloadDocumentWithMetaData`